TaskService/App_Start/OpenIdConnectCachingSecurityTokenProvider.cs

```diff
@@ -17,16 +13,11 @@ namespace TaskService.App_Start
     public class OpenIdConnectCachingSecurityTokenProvider : IIssuerSecurityTokenProvider
     {
         public ConfigurationManager<OpenIdConnectConfiguration> _configManager;
-        private string _issuer;
-        private IEnumerable<SecurityToken> _tokens;
-        private readonly string _metadataEndpoint;
-
-        private readonly ReaderWriterLockSlim _synclock = new ReaderWriterLockSlim();
 
         public OpenIdConnectCachingSecurityTokenProvider(string metadataEndpoint)
         {
-            _metadataEndpoint = metadataEndpoint;
-            _configManager = new ConfigurationManager<OpenIdConnectConfiguration>(metadataEndpoint);
+            HttpClient httpClient = new HttpClient();
+            _configManager = new ConfigurationManager<OpenIdConnectConfiguration>(metadataEndpoint, httpClient);
```

Comment:
nit: you could pass new HttpClient() as a parameter and not have the local variable.